app/views/common/_pagination.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="form-group">
+  <%=
+
+    select_tag('per_page',
+                 options_for_select(per_page_options || Setting[:entries_per_page]),
+                 label: _('per page'),
+                 onChange: "tfm.tools.changePerPageNumber()",
+                 class: "pagination-pf-pagesize without_select2") %>
+  <span><%= _('per page') %></span>
+</div>
+
+<div class="form-group">
+  <span>
+    <span class='pagination-pf-items-current'>
+      <%= "%{from}-%{to}" %
+              { :from => collection.offset + 1, :to => collection.offset + collection.length } %>
```

Comment:
You can get rid of the string interpolation here using something like:
`<%= collection.offset + 1 %>-<%= collection.offset + collection.length %>`